src/core/formatters.ts

```diff
@@ -131,6 +131,51 @@ export module Formatters {
   }
 
   /**
+   * Creates a formatter for values that displays abbreviated values
+   * and uses standard short scale suffixes (thousands, millions, billions, trillions,
+   * quadrillions).
```

Comment:
Decently minor: Can you put the "thousands, millions, etc." part on a separate line?  I personally don't like that "quadrillions" just gets its own line.

Also, it'd be more clear that we only satisfy up to quadrillions.